### PR TITLE
test(core): speed up tests

### PR DESCRIPTION
**Pull request by jerrinot**

### CODE REVIEW

**Comment by jerrinot** [comment]
also a failure due to too short spin lock timeout in some tests

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 8 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/std/str/Utf8StringSink.java|8|8|100.00%|

File: core/src/main/java/io/questdb/std/str/Utf8StringSink.java [hunk 1]
```diff
@@ -160,7 +166,7 @@ public int size() {
     private void checkCapacity(int extra) {
         assert extra >= 0;
         int size = pos + extra;
-        if (buffer.length > size) {
+        if (buffer.length >= size) {
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/std/str/Utf8StringSink.java:169]: what prompted this change?

File: core/src/main/java/io/questdb/std/str/Utf8StringSink.java [hunk 2]
```diff
@@ -175,4 +181,13 @@ private Utf8StringSink putByte0(byte b) {
         buffer[pos++] = b;
         return this;
     }
+
+    @NotNull
+    private Utf8StringSink putByte0Repeat(byte b, int n) {
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/std/str/Utf8StringSink.java:186]: ignored warning, unused return value

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 7 / 7 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/std/str/Utf8StringSink.java|7|7|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**